src/components/VHeader/VHeaderDesktop.vue

```diff
@@ -7,7 +7,7 @@
         : 'border-white'
     "
   >
-    <VLogoButton :is-fetching="isFetching" />
+    <VLogoButton :is-fetching="isFetching" class="h-12" />
```

Comment:
Is there a reason for this to be here instead of inside of the VLogoButton component? Is this essentially an override for a very specific use of the button?